### ISSUE TITLE
Kartlag for datakilder

### ISSUE THREAD

**Issue by svetlnaumova** [Artsdatabanken/nin-kart-frontend#1549]
Når man navigerer til datakilder og videre til Artsdatabanken eller andre dataleverandører og deretter til fanen "Kart", ser det ut som det skal være et eget kartlag for denne dataleverandøren (se screenshot). Burde kanskje rettes opp?
![03](https://user-images.githubusercontent.com/43623201/65774048-cd972480-e13d-11e9-90a0-3f0ac0b70b5a.jpg)

**Comment by helemork** [comment]
Hvordan ønsker vi dette skal vises frem?
relatert til #1388

**Comment by bjornreppen** [comment]
Kunne vi vist kartlagene som publiseres av kilden i lista, og eventuelt et kart som viser hvilke områder som dekkes av dataene?

**Comment by helemork** [comment]
På våre egen side og milj.dir vil det jo bli ganske mye isåfall (så fort milj.dir dataene er tilbake), så usikker på hvordan det kan fikses på en fornuftig måte?

**Comment by bjornreppen** [comment]
Hvis man bare viser toppnivået i datasettet, eksempel
- **NINA**: Regional naturvariasjon
- **Artsdatabanken**: Arter
- **Miljødirektoratet:** Natursystem, Verneområder

Kartet kunne for eksempel vise et enkelt dekningskart for datasettet = hvor finnes det data

**Comment by helemork** [comment]
Liker det. Men - det burde jo vise det samme som vi viser på øverste node andre steder.
ulempen da er kanskje for mange farger...

**Comment by bjornreppen** [comment]
Ja, det var nok hakket bedre..  Ja, hvis vi skal liste alle miljøvariabler under Miljødirektoratet blir det mange... et symptom kanskje på at de burde vært gruppert bedre?
